tools/etcd-dump-logs/main.go

```diff
@@ -51,7 +52,8 @@ const (
 func main() {
 	snapfile := flag.String("start-snap", "", "The base name of snapshot file to start dumping")
 	waldir := flag.String("wal-dir", "", "If set, dumps WAL from the informed path, rather than following the standard 'data_dir/member/wal/' location")
-	index := flag.Uint64("start-index", 0, "The index to start dumping")
+	startIndex := flag.Uint64("start-index", 0, "The index to start dumping (exclusive)")
```

Comment:
Oh start index being exclusive is weird, but looks like it was implemented like this originally